packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -374,10 +377,34 @@ export class LionInputDatepicker extends ScopedElementsMixin(
     }
     if (this._syncOnUserSelect) {
       // Synchronize new selectedDate value to input
+      this._isCalendarTriggeredByUser = true;
```

Comment:
Nice improvement 👍 

Could you configure the parent FormatMixin here (use variable `_isHandlingUserInput` instead of `_isCalendarTriggeredByUser`)?

Then we can also leave out the _dispatchModelValueChangedEvent, because it uses the one from FormatMixin

https://github.com/ing-bank/lion/blob/2b73c50f6e7ff9ef96122a74536e7c4df9b8553b/packages/form-core/src/FormatMixin.js#L307

Reply:
I tried, but it broke the input. The test worked, but the value didn't get displayed in the demo.